vng/vector/primitive.go

```diff
@@ -213,9 +212,10 @@ func (p *PrimitiveReader) ReadBytes() (zcode.Bytes, error) {
 func (p *PrimitiveReader) next() error {
 	segment := p.segmap[0]
 	p.segmap = p.segmap[1:]
-	if segment.Length > 2*MaxSegmentThresh {
-		return errors.New("corrupt VNG: segment too big")
-	}
+	//TODO Segments are currently disabled.
+	//if segment.Length > 2*MaxSegmentThresh {
+	//    return errors.New("corrupt VNG: segment too big")
+	//}
```

Comment:
I don't see anything failing without the changes in this file. Unless I'm missing something, let's leave it as is.

Reply:
On main:

```
> zq -f vng -o ints.vng 'yield this, this' ints.zng                                                      

> zq 'count()' ints.vng                                                                                  
ints.vng: corrupt VNG: segment too big
```

You changed the write path to make one big segment, but the read path still panics if the segment is over the max length.